src/demo-app/paginator/paginator-demo.scss

```diff
@@ -1,7 +1,6 @@
 .demo-section {
   max-width: 500px;
   margin-bottom: 24px;
-  background: #efefef !important;
```

Comment:
The demos are supposed to be functional, rather than pretty. I think the idea with the `background` in this case is to be able to distinguish the paginator from the card.

Reply:
Yeah, just to clarify, @gerardob, everything in "demo-app" is just for our own development/testing. We don't publish it anywhere.

Reply:
ok, I understand is good to know! thanks guys!